OpenRA.Mods.Cnc/Traits/SpawnViceroid.cs

```diff
@@ -33,7 +33,7 @@ class SpawnViceroid : INotifyKilled
 		public void Killed(Actor self, AttackInfo e)
 		{
 			if (!self.World.LobbyInfo.GlobalSettings.Creeps) return;
-			if (e.Warhead == null || e.Warhead.DeathType != spawnViceroidInfo.DeathType) return;
+			if (e.Warhead == null || !e.Warhead.DamageTypes.Contains(spawnViceroidInfo.DeathType)) return;
```

Comment:
It would be great if you could rename `spawnViceroidInfo` &rarr; `info` while you are in this file.